sphinx/source/docs/releases/0.13.0.rst

```diff
@@ -15,6 +15,13 @@ Migration Guide
 NOTE: the 0.13.x series is a final series of small releases leading to a
 1.0 release. For more information see the `project roadmap`_.
 
+Max Websocket Message Size
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A new command line option ``--websocket-max-message-size`` for the Bokeh
+server can be used to configure the Tornado ``websocket_max_message_size``
+option. The default value is 20mb.
```

Comment:
The default value is *now* 20mb (was previously defaulting to 10mb)?

Reply:
(Seems like that's what a user would want to know, i.e. how the value has changed.)